packages/client-core/src/components/Glass/ToolbarAndSidebar.tsx

```diff
@@ -118,7 +118,7 @@ const backButtonStyles = `
 const Tab = ({ onClick, heading, active }: TabProps) => {
   return (
     <button className={`group`} onClick={onClick}>
-      <h2 className={`lg:text-shadow-md grid gap-y-1`}>
+      <h2 className={`lg:text-shadow-md text-shadow grid gap-y-1 font-dm-sans text-base md:text-4xl`}>
```

Comment:
I think this `md` should be a `lg`, otherwise its causing these styles on `md`

![image](https://github.com/user-attachments/assets/0f49d382-b956-4db5-994a-7c56c67b5e20)